packages/multichain-transactions-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/multichain-transactions-controller",
-  "version": "0.6.0",
+  "version": "1.0.0",
```

Comment:
Can you make it a `0.7.0` instead? We're close to stabilize this new controller, but for now we keep it under <1.0.0.

Reply:
Done.